docs/taskbackground.rst

```diff
@@ -0,0 +1,109 @@
+Physics Analysis Background
+===============================================================
+
+:math:`t\bar{t}` Events
+---------------------------------------------------------------
+The Analysis Grand Challenge aims to demonstrate the integration of different software components in a realistic physics analysis. The task chosen for CMS open data is a measurement of the :math:`t\bar{t}` (top-antitop quark pair production) crosssection  measurement using events with at least four jets and one lepton in the final state. This signature has two possibilities, depending on whether the lepton/neutrino pair originates from the top or antitop quark:
```

Comment:
Let's slightly change the first sentence to something like

> Implementations of analysis tasks in the Analysis Grand Challenge aim to demonstrate...

as the AGC itself is broader than that.